test/e2e/e2e.bats

```diff
@@ -2,25 +2,32 @@
 
 source test/e2e/helpers.sh
 
-@test "shp binary can be executed" {
-	result="$(shp --help)"
-	[ ! -z "$result" ]
+setup() {
+	load 'bats/support/load'
+	load 'bats/assert/load'
+	load 'bats/file/load'
+}
+
+teardown() {
+	run kubectl delete builds.shipwright.io --all
+	run kubectl delete buildruns.shipwright.io --all
 }
 
-@test "shp build create succesfully run with 1 argument" {
-	build_name=$(random_name)
-	result="$(shp build create ${build_name} --source-url=url --output-image=image)"
-        [ ! -z "$result" ]
-	result=$(kubectl get build.shipwright.io ${build_name})
-	[ ! -z "$result" ]
+@test "shp binary can be executed" {
+	run shp --help
+	assert_success
 }
 
-@test "shp buildrun create succesfully run with 1 argument" {
-        build_name=$(random_name)
-        result="$(shp build create ${build_name} --source-url=url --output-image=image)"
-        [ ! -z "$result" ]
+@test "shp [build/buildrun] create should not error when a name is specified" {
+	# generate random names for our build and buildrun
+    build_name=$(random_name)
 	buildrun_name=$(random_name)
-        result="$(shp buildrun create ${buildrun_name} --buildref-name={$build_name})"
-	result=$(kubectl get buildrun.shipwright.io ${buildrun_name})
-        [ ! -z "$result" ]
+
+	# ensure that shp build create does not give an error when a build_name is specified
+    run shp build create ${build_name} --source-url=url --output-image=image
+    assert_success
+
+	# ensure that shp buildrun create does not give an error when a buildrun_name is specified
+    run shp buildrun create ${buildrun_name} --buildref-name={$build_name}
```

Comment:
There's a typo on the `--buildref-name` curly brace position. It should look like the following:

```bash
run shp buildrun create ${buildrun_name} --buildref-name=${build_name}
```